FlowCrypt/Functionality/Mail Provider/EmptyInboxProvider/Gmail+EmptyInbox.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Gmail+EmptyInbox.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 6/28/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import FlowCryptCommon
+import GoogleAPIClientForREST_Gmail
+
+extension GmailService: EmptyInboxProvider {
+    func emptyFolder(path: String) async throws {
+        let context = FetchMessageContext(folderPath: path, count: nil, pagination: nil)
+        let list = try await fetchMessagesList(using: context)
```

Comment:
@ioanmo226 will it fetch only first page of messages in the folder or full folder messages list?